bgpd/bgp_bmp.c

```diff
@@ -1252,19 +1246,16 @@ static bool bmp_wrqueue_locrib(struct bmp* bmp, struct pullwr* pullwr) {
 		break;
 	}
 
-	zlog_info("passed afi state check");
-
 	peer = QOBJ_GET_TYPESAFE(bqe->peerid, peer);
 	if (!peer) {
-		zlog_info("bmp: skipping queued item for deleted peer");
+		// skipping queued item for deleted peer
 		goto out;
```

Comment:
'//' is notation for C++. please  use '/* */'

Reply:
this type of comments was already used in the original BMP implementation
do you want me to change my added comments to `/* */` anyways ?

Reply:
changed in [dd6bf8a](https://github.com/FRRouting/frr/commit/dd6bf8ade441007923e83f078cbdedbad5a273ff)